e2e/kind-config.yaml

```diff
@@ -1,6 +1,8 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 name: e2e
+networking:
+  ipFamily: dual
```

Comment:
Please also extend the README for running the e2e tests, as this now requires having IPv6 enabled.